File: leetcode/2457/2457.scala
object Solution 
{
    def unequalTriplets(nums: Array[Int]): Int = 
    {
        var len = nums.length
        var ans = 0
        for (i <- 0 until len) 
        {
            for (j <- i + 1 until len) 
            {
                for (k <- j + 1 until len) 
                {
                    if (nums(i) != nums(j) && nums(j) != nums(k) && nums(i) != nums(k)) 
                    {
                        ans += 1
                    }
                }
            }
        }
        ans
    }
}

File: leetcode/2457/2457.java
class Solution 
{
    public int unequalTriplets(int[] nums)
    {
        int len = nums.length;
        int ans = 0;
        for(int i = 0;i<len;i++)
        {
            for(int j = i+1;j<len;j++)
            {
                for(int k = j+1;k<len;k++)
                {
                    if(nums[i] != nums[j] && nums[j] != nums[k] && nums[i] != nums[k])
                    {
                        ans++;
                    }
                }
            }
        }
        return ans;

    }
}